src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsEvoSuiteComponent.kt

```diff
@@ -38,27 +39,26 @@ class TestGenieSettingsComponent {
 
     init {
         panel = FormBuilder.createFormBuilder()
-            .addComponent(JXTitledSeparator("General settings"))
-            .addComponent(showCoverageCheckBox, 10)
-            .addComponent(sandboxCheckBox, 10)
-            .addComponent(assertionsCheckBox, 10)
-            .addLabeledComponent(JBLabel("Seed(random if left empty) "), seedTextField, 10, false)
-            .addLabeledComponent(JBLabel("Select search algorithm"), algorithmSelector, 10, false)
-            .addLabeledComponent(JBLabel("Select configuration id (null if left empty) "), configurationIdTextField, 5, false)
-            .addComponent(clientOnThreadCheckBox, 10)
-            .addComponent(minimizeCheckBox, 10)
-            .addComponent(junitCheckCheckBox, 10)
-            .addComponent(criterionSeparator, 15)
-            .addComponent(criterionLineCheckBox, 5)
-            .addComponent(criterionBranchCheckBox, 5)
-            .addComponent(criterionExceptionCheckBox, 5)
-            .addComponent(criterionWeakMutationCheckBox, 5)
-            .addComponent(criterionOutputCheckBox, 5)
-            .addComponent(criterionMethodCheckBox, 5)
-            .addComponent(criterionMethodNoExceptionCheckBox, 5)
-            .addComponent(criterionCBranchCheckBox, 5)
-            .addComponentFillVertically(JPanel(), 0)
-            .panel
+                .addComponent(JXTitledSeparator("General settings"))
+                .addLabeledComponent(JBLabel("Select search algorithm"), algorithmSelector, 10, false)
```

Comment:
On our pull request for Quick Access parameters, Marti suggested me separating a similar blob in the constructor with comments (for each category) and add a blank line before a comment block, since it improves readability. Is it worth to do the same thing here?

Reply:
Will assess this

Reply:
Yeah, i very much like this way